### ISSUE TITLE
get_dofs broken in 06_elasticity

### ISSUE THREAD

**Issue by gdmcbain** [gdmcbain/fenics-tuto-in-skfem#8]
https://github.com/gdmcbain/fenics-tuto-in-skfem/blob/f10fc315e28d8677930e6c7257c2773cd6f6a8cf/06_elasticity/st06_elasticity.py#L27

indirectly raises

```
TypeError: <lambda>() missing 2 required positional arguments: 'y' and 'z'

**Comment by gdmcbain** [comment]
This is kinnala/scikit-fem#51.

**Comment by gdmcbain** [comment]
Will be fixed by fa580499ed85e18b9785e3d489b96664ae113e70.

**Comment by gdmcbain** [comment]
Fixed by #9.
